performance_manager/pyproject.toml

```diff
@@ -0,0 +1,21 @@
+[tool.poetry]
```

Comment:
dependencies for performance manager application. i started with things based on postgres reading and the notebook @rymarczy posted.